CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+# dbt_iterable_source v0.8.0
+
+## API Updates
+[PR #28](https://github.com/fivetran/dbt_iterable_source/pull/28) includes updates in response to the [Aug 2023 updates](https://fivetran.com/docs/applications/iterable/changelog#august2023) for the Iterable connector.
+
+For customers using the new August 2023 schema, please note the following changes:
+-  We have removed `user_device_history` as the value is marginal.
```

Comment:
This is actually a breaking change and should categorized as such. I like that it is right at the top, but let's add this to a breaking changes section.

Reply:
Forgot to add that breaking changes title-- updated